docs/source/en/using-diffusers/controlling_generation.mdx

```diff
@@ -146,3 +147,25 @@ See [here](../training/dreambooth) for more information on how to use it.
 [Textual Inversion](../training/text_inversion) fine-tunes a model to teach it about a new concept. I.e. a few pictures of a style of artwork can be used to generate images in that style.
 
 See [here](../training/text_inversion) for more information on how to use it.
+
+## MultiDiffusion Panorama
+
+[Paper](https://multidiffusion.github.io/)
+[Demo](https://huggingface.co/spaces/weizmannscience/MultiDiffusion)
+MultiDiffusion defines a new generation process over a pre-trained diffusion model. This process binds together multiple diffusion generation processes can be readily applied to generate high quality and diverse images that adhere to user-provided controls, such as desired aspect ratio (e.g., panorama), and spatial guiding signals, ranging from tight segmentation masks to bounding boxes.
+[MultiDiffusion Panorama](../api/pipelines/stable_diffusion/panorama) allows to generate high-quality images at arbitrary aspect ratios (e.g., panoramas).
+
+See [here](../api/pipelines/stable_diffusion/panorama) for more information on how to use it to generate panoramic images.
```

Comment:
If you rebase with `main` this shouldn't be needed. Or am I missing out on anything?

Reply:
Fixed in https://github.com/huggingface/diffusers/pull/2407/commits/3981459a6a98951f56a414c0247ff4488ce40659